README.md

```diff
@@ -352,25 +352,38 @@ Validator nodes proceed with the exact reverse process as that used by the seque
 Currently, there are two commitment modes supported with unique encoding schemas for each. The `version byte` is shared for all modes and denotes which version of the EigenDA `DACertificate` is being used/requested. The following versions are currently supported:
 * `0x00`: EigenDA V1 certificate type (i.e, dispersal blob info struct with verification against service manager)
 * `0x01`: EigenDA V2 certificate type
+* `0x02`: EigenDA V2 certificate type with encoding byte
+
+For certificates with version byte 0x02 and higher, an encoding byte is included that specifies the serialization format:
+* `0x00`: RLP encoding (default)
+* `0x01`: `EigenDACertVerifier` *verifyDACertV2* ABI encoding (for smart contract verification)
 
 #### Optimism Commitment Mode
 For `alt-da` Optimism rollups using EigenDA, the following [commitment schemas](https://specs.optimism.io/experimental/alt-da.html#example-commitments) are supported by our proxy:
 
-| commitment_type (byte) | da_layer_byte | version_byte | payload           |
-| ---------------------- | ------------- | ------------ | ----------------- |
-| 0x00                   |               |              | keccak_commitment |
-| 0x01                   | 0x00          | 0x00         | eigenda_cert_v1   |
-| 0x01                   | 0x00          | 0x01         | eigenda_cert_v2   |
+| commitment_type (byte) | da_layer_byte | version_byte | encoding_byte | payload           |
+| ---------------------- | ------------- | ------------ | ------------- | ----------------- |
+| 0x00                   |               |              |               | keccak_commitment |
+| 0x01                   | 0x00          | 0x00         | N/A           | eigenda_cert_v0   |
+| 0x01                   | 0x00          | 0x01         | N/A           | eigenda_cert_v1   |
```

Comment:
I'd personally remove the N/A and follow the pattern we had previously - just empty slot when its not present. Find its easier to parse as opposed to a diagram cluttered with N/As

Reply:
04dba8c7e4368cfd65fdfc4b9bb95f480a8ec008